lib/ex_aws/s3/utils.ex

```diff
@@ -111,6 +111,7 @@ defmodule ExAws.S3.Utils do
     |> IO.iodata_to_binary
   end
 
+  def normalize_param(:version_id), do: "versionId"
```

Comment:
Annotation: This parameter needs to be `camelCase` unlike all of the other ones which are `kebab-case`.  Link to docs.

Example in the AWS [docs](https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObject.html) is:
```
GET /myObject?versionId=3/L4kqtJlcpXroDTDmpUMLUo HTTP/1.1
Host: bucket.s3.amazonaws.com
Date: Wed, 28 Oct 2009 22:32:00 GMT
Authorization: authorization string
```